src/main/resources/db/migration/V010__Add_shedlock.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE shedlock(
+    name VARCHAR(64),
+    lock_until TIMESTAMP(3) NULL,
+    locked_at TIMESTAMP(3) NULL,
+    locked_by  VARCHAR(255),
```

Comment:
what's the default? `NULL` or `NOT NULL`?

Reply:
This is step by step follow up of [README](https://github.com/lukas-krecan/ShedLock#jdbctemplate). Exactly same thing was applied in bulk scan project too